README.md

```diff
@@ -1 +1,24 @@
-# sea-map
+# SEA-MAP Tech vs. Abuse project
+
+### Site URLs
+- [Alpha](http://neontribe.github.io/sea-map)
+
+### Development 
```

Comment:
Should this be ## and the headings beneath ### ?

Reply:
lol. Yes. I need your plugin!

Reply:
😅 https://github.com/JamshedVesuna/vim-markdown-preview maybe